data/libs/Game.lua

```diff
@@ -9,8 +13,31 @@ Game.GetCommsLines = function()
 	return Game.comms_log_lines
 end
 
+-- Function: GetStartTime()
+--
+-- Returns the zero-offset time in seconds since Jan 1 3200 at which the
+-- current game began
```

Comment:
Should one explain why player might need it? I.e. a new game doesn't start the timer on 0? Maybe implicitly understood, but my lazy brain would miss it. I.e. an example perhaps? 
```lua
--
-- > local seconds_since_start = Game.time() - Game.GetStartTime()
--
```

Should one also mention that it's _in game_ time?

Reply:
I don't consider it high-priority as the name is relatively self-documenting, but I'll add that usage line.